src/components/Textarea/Textarea.tsx

```diff
@@ -52,6 +63,9 @@ export const Textarea = forwardRef<HTMLTextAreaElement, TextareaProps>(
       active,
       typed,
     } = useStateEvents(value, onChange);
+    const textAreaRef = useRef<HTMLTextAreaElement>(null);
+    useAutosizeTextArea(textAreaRef.current, value, autoSize);
+    useImperativeHandle(ref, () => textAreaRef.current!);
```

Comment:
**Praise:** good job - TIL that there is [useImperativeHandle](https://react.dev/reference/react/useImperativeHandle) 👏🏻